docs/src/pages/docs/Connecting to Databases/index.mdx

```diff
@@ -29,7 +29,7 @@ A list of some of the recommended packages.
 |[Apache Hive](/docs/databases/hive)|```pip install pyhive```|```hive://hive@{hostname}:{port}/{database}```|
 |[Apache Impala](/docs/databases/impala)|```pip install impala```|```impala://{hostname}:{port}/{database}```|
 |[Apache Kylin](/docs/databases/kylin)|```pip install kylinpy```|```kylin://<username>:<password>@<hostname>:<port>/<project>?<param1>=<value1>&<param2>=<value2>```|
-|[Apache Pinot](/docs/databases/pinot)|```pip install pinotdb```|```pinot+http://CONTROLLER:5436/ query?server=http://CONTROLLER:5983/```|
+|[Apache Pinot](/docs/databases/pinot)|```pip install pinotdb```|```pinot+http://BROKER:5436/query?server=http://CONTROLLER:5983/```|
```

Comment:
reading the Pinot docs, it looks like maybe this should be `pinot://` instead of `pinot+http://`, do you think that change should be made here too?

Reply:
I strongly second this, we should drop the `+http`

Reply:
yes drop the +http